pytorch_lightning/utilities/imports.py

```diff
@@ -54,13 +54,13 @@ def _compare_version(package: str, op, version) -> bool:
     except (ModuleNotFoundError, DistributionNotFound):
         return False
     try:
-        pkg_version = LooseVersion(pkg.__version__)
+        pkg_version = Version(pkg.__version__)
```

Comment:
Can you add some tests for the version numbers that require these changes?

Reply:
not really, this function is loading the version from the system so patch/mock does not work...
```python
@mock.patch("torch.__version__", return_value="1.8.0a19")
def test_compare_version(torch):
    assert _compare_version("torch", operator.ge, "1.7.1")
    assert not _compare_version("torch", operator.ge, "1.8.0")
```
cc: @awaelchli